### PR TITLE
update to node 16

### PR DESCRIPTION
**Pull request by chmelevskij**
Update to node 16

### CODE REVIEW

**Review by Asizon — APPROVED** [review]
Cool!!

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
Should this be for 10.8?
We should keep node version the same across Configurator and Log Viewer, and not change late in the dev process unless it is to fix a bug.

**Comment by haslinghuis** [comment]
According to @Asizon it's just a build improvement, already provided a PR for Blackbox to synchronize.

File: .nvmrc
Comment (haslinghuis) [inline .nvmrc:1]:
```suggestion
v16.15.0
```

File: .travis.yml
Comment (haslinghuis) [inline .travis.yml:25]:
```suggestion
  - 16.15.0
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Please update to v16.15.0 in remaining files.

**Comment by haslinghuis** [comment]
Superseded by https://github.com/betaflight/betaflight-configurator/pull/2913

**Pull request closed without merging**
